src/styles/section.scss

```diff
@@ -23,6 +23,10 @@
   display: flex;
   align-items: center;
   justify-content: center;
+
+  & > *:not(:last-child) {
+    margin-right: $size-8;
+  }
```

Comment:
This is quite specific, meaning difficult to override.

Might be better to add another class and move these styles into that class and not make it nested.